test/BUILD

```diff
@@ -1,10 +1,73 @@
 package(default_visibility=["//visibility:public"])
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
+load("@build_bazel_rules_nodejs//:defs.bzl", "jasmine_node_test")
 
 ts_library(
-    name = "test",
-    srcs = glob(["*.ts"]),
+    name = "test_support",
+    srcs = ["test_support.ts"],
     deps = ["//src"],
     tsconfig = "//:tsconfig.json",
-    testonly = 1,
 )
+
+ts_library(
+    name = "unit_test_lib",
+    srcs = [
```

Comment:
any thoughts yet about re-using this build file in google3 after this PR is synced? hopefully this would be the only place these files are listed

Reply:
Not yet, but that should be possible.  I've recently spent some time improving the tsickle sync proces.